test/test_server_upload.py

```diff
@@ -33,6 +53,7 @@ class MockHandler(object):
     def __init__(self, host='notebook-server:8888', protocol='http'):
         self.request = MockRequest(host, protocol)
         self.last_redirect = None
+        self.tools = jupyter_cms.bundler.BundlerTools()
```

Comment:
I'm still iffy on whether we want the CMS bits here or not. The tools part is only needed for the associated files. But that's already tested in `jupyter_cms` unit tests. More important here is checking whether the decl widgets get packaged or not when a notebook is using them. (Clearly, we don't have good test coverage for it since I manually hit a long standing bug on my first try.)